src/server/server.js

```diff
@@ -49,6 +49,14 @@ const numProcesses = process.env.NUM_PROCESSES || os.cpus().length;
 
 app.use(requestTime(numProcesses));
 
+// drop set-cookie headers for everything but state api
+app.use(function*(next) {
+    yield* next;
+    if (this.request.url !== '/api/v1/state') {
```

Comment:
Please add exceptions for user sign up and account recovery routes. Look for them in account_recovery.js and enter_confirm_email/phone.js

Reply:
Good catch, I enabled set-cookie headers on everything that looks like it could be an entry point now